modin/pandas/dataframe.py

```diff
@@ -965,10 +965,46 @@ def convert_objects(
         )
 
     def corr(self, method="pearson", min_periods=1):
-        raise NotImplementedError(
-            "To contribute to Pandas on Ray, please visit "
-            "github.com/modin-project/modin."
-        )
+        """Compute pairwise correlation of columns, excluding NA/null values.
```

Comment:
Please move the logic for computing the `corr` and `cov` to the `DataManager` class.

This class will call `self._data_manager.corr(...)`, etc.

Reply:
Moved the logic, let me know if there's any other changes that need to be made!